src/pages/converter/index_page.cr

```diff
@@ -3,28 +3,49 @@ class ConvertHtmlToLucky::IndexPage < PageLayout
   needs output : String
 
   def content
-    div class: "my-4" do
+    div class: "my-4 mx-2" do
       h2 @title, class: "text-3xl mb-4"
       para "Lucky uses Crystal classes and methods to generate HTML. It may sound crazy at first, but the advantages are numerous. Never accidentally print nil to the page, extract and share partials using regular methods. Easily read an entire page by looking at just the render method. Text is automatically escaped for security. And it’s all type safe. That means no more unmatched closing tags, and never rendering a page with missing data."
 
-      div class: "flex my-4 space-x-4 items-stretch" do
-        div class: "w-1/2" do
+      form_for ConvertHtmlToLucky::Create, class: "my-4 space-y-2" do
```

Comment:
It felt like this whole content section is the "form", though technically this could end right after the `render_convert_button` call if we want to exclude the output.